src/idb.filesystem.js

```diff
@@ -34,16 +34,17 @@ if (exports.requestFileSystem || exports.webkitRequestFileSystem) {
 }
 
 // Bomb out if no indexedDB available
-const indexedDB = exports.indexedDB || exports.mozIndexedDB ||
+var indexedDB = exports.indexedDB || exports.mozIndexedDB ||
                   exports.msIndexedDB;
 if (!indexedDB) {
   return;
 }
 
-let IDB_SUPPORTS_BLOB = true;
+var IDB_SUPPORTS_BLOB = false;
```

Comment:
I love led and const, but it seems to not make a difference and some people (like me) like to use the not minified files in projects, which then would produce an error.